README.md

```diff
@@ -20,7 +24,7 @@ Add the SDK as a dependency in your project.
 </dependency>
 ```
 
-### 2. Download Jar File
+### Install By Downloading the Jar File
```

Comment:
Please add a link to the sonatype library page. Best I can do is: https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22smartsheet-sdk-java%22
Direct download of a JAR feels dangerous and abrupt, so we should provide an alternative as well.